### DISCUSSION TITLE
Sveltekit Strapi Blog

### DISCUSSION THREAD

**Discussion by Marktawa** [Marktawa/sveltekit-strapi-blog#2] [Announcements]
# Sveltekt Strapi Blog

![Cover Image](https://res.cloudinary.com/craigsims808/image/upload/v1708262727/articles/sveltekit-strapi-blog/cover_pcy8tr.png)

## **Part 01: Setup and Installation**
Status: Complete

## Project Description

This is a blog website built using Strapi as a headless CMS and Sveltekit as the frontend framework.

## Article Link

[Build a blog from scratch using Strapi and Sveltekit]()
 
## Prerequisites

- Node.js (LTS version)

## Getting Started

Unzip source code.
```bash
unzip sveltekit-strapi-blog.zip
```
Change directory.
```bash
cd sveltekit-strapi-blog
```

## Strapi Setup

Install dependencies.
```bash
cd backend
npm install
```

Build your Strapi app.
```bash
npm run build
```

Start your Strapi dev server.
```bash
npm run develop
```

Visit [localhost:1337/admin](http://localhost:1337/admin) in your browser to view the Strapi Admin Dashboard.

## Sveltekit Setup

Open up a new terminal session and open up the main project folder, `sveltekit-strapi-blog`. 

Install dependencies.
```bash
cd frontend
npm install
```

Run Sveltekit app.
```bash
npm run dev
```

Visit [localhost:5173](http://localhost:5173) in your browser to view the Sveltekit app.

## Author

[Mark Munyaka](https://markmunyaka.com)

GitHub: [@Marktawa](https://github.com/Marktawa)
Twitter: [@McMunyaka](https://twitter.com/McMunyaka)

## Sponsor

Support my passion for sharing development knowledge by making a donation to my [**Buy Me a Coffee**](https://www.buymeacoffee.com/markmunyaka) account. Your contribution helps me create valuable content and resources. Thank you for your support!

[![Buy Me A Coffee Banner](https://res.cloudinary.com/craigsims808/image/upload/v1708089939/articles/test/buymeacoffee_lqmwjn.png)](https://www.buymeacoffee.com/markmunyaka)

[Buy Me A Coffee](https://www.buymeacoffee.com/markmunyaka)

## What's Changed
* Part01 by @Marktawa in https://github.com/Marktawa/sveltekit-strapi-blog/pull/1

## New Contributors
* @Marktawa made their first contribution in https://github.com/Marktawa/sveltekit-strapi-blog/pull/1

**Full Changelog**: https://github.com/Marktawa/sveltekit-strapi-blog/commits/v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Marktawa/sveltekit-strapi-blog/releases/tag/'>Sveltekit Strapi Blog</a>.</em>

**Comment by craigsims808** [comment]
Looks good!
